### PR TITLE
open-cluster-management/backlog#4748 KI vm dest

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -221,6 +221,17 @@ You must be logged in as `root` to run the `management-ingress` service.
 
 Search maps RBAC for resources in the hub cluster. Depending on user RBAC settings for {product-title-short}, users might not see node data from the managed cluster. Results from search might be different from what is displayed on the _Nodes_ page for a cluster.
 
+[#vmware-cluster-might-not-be-destroyed]
+=== Process to destroy a vmWare vSphere cluster does not complete
```
Comment (chrisahl) [inline release_notes/known_issues.adoc]: vmWare should be VMware

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -221,6 +221,17 @@ You must be logged in as `root` to run the `management-ingress` service.
...
+
+// 2.1.0:4748
+
+When you destroy a vmWare vSphere managed cluster, the status continues to display `Destroying`, and the cluster is not destroyed. To resolve this issue, manually remove the namespace of the managed cluster by running the following command:
```
Comment (chrisahl) [inline release_notes/known_issues.adoc]: vmWare should be VMware

**Review by chrisahl — CHANGES_REQUESTED** [review]
comments inline

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (cadawson) [reply release_notes/known_issues.adoc]: Geez, yeah. Too early.

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (cadawson) [reply release_notes/known_issues.adoc]: Thanks

**Comment by cadawson** [comment]
Let's try that again @chrisahl . Thanks!

**Comment by cadawson** [comment]
For some reason that didn't save the change. Now it is in @chrisahl .

**Review by chrisahl — APPROVED** [review]

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -221,6 +221,17 @@ You must be logged in as `root` to run the `management-ingress` service.
 
 Search maps RBAC for resources in the hub cluster. Depending on user RBAC settings for {product-title-short}, users might not see node data from the managed cluster. Results from search might be different from what is displayed on the _Nodes_ page for a cluster.
 
+[#vmware-cluster-might-not-be-destroyed]
+=== Process to destroy a VMware vSphere cluster does not complete
+
+// 2.1.0:4748
+
+When you destroy a VMware vSphere managed cluster, the status continues to display `Destroying`, and the cluster is not destroyed. To resolve this issue, manually remove the namespace of the managed cluster by running the following command:
```
Comment (chrisahl) [inline release_notes/known_issues.adoc]: Based on a recent bug found in Azure, we need to make a few edits to this. 
```
[#cluster-might-not-be-destroyed]
=== Process to destroy a cluster does not complete

// 2.1.0:4748

When you destroy a managed cluster, the status continues to display `Destroying` after one hour, and the cluster is not destroyed. To resolve this issue:
- manually ensure all the provider resources associated with the managed cluster are cleaned up  
- manually remove the namespace of the managed cluster by running the following command:
```

**Comment by chrisahl** [comment]
@TheRealHaoLiu This is the doc item for VMware cleanup when deprovisioning gets stuck.   Add info you want doc'ed

**Pull request merged**